tests/sql/close.yaml

```diff
@@ -0,0 +1,53 @@
+$schema: ../../json_schemas/test_story.schema.yaml
+
+description: Test to explicitly clear the cursor context
+
+prologues:
+  - path: /{index}
+    method: PUT
+    parameters:
+      index: books
+    request_body:
+      payload: {}
+epilogues:
+  - path: /books
+    method: DELETE
+    status: [200, 404]
+chapters:
+  - synopsis: Create few documents.
+    path: /_bulk
+    method: POST
+    request_body:
+      content_type: application/x-ndjson
+      payload:
+        - {create: {_index: books, _id: book_1392214}}
+        - {author: Harper Lee, title: To Kill a Mockingbird, year: 1960}
+        - {create: {_index: books, _id: book_1392215}}
+        - {author: Elizabeth Rudnick, title: Beauty and the Beast, year: 1991}
+  - synopsis: Refresh the index.
+    path: /{index}/_refresh
+    method: POST
+    parameters:
+      index: books
+    response:
+      status: 200
```

Comment:
The above should be moved to a prologue because they aren't exercising the SQL API namespace.